### ISSUE TITLE
search question

### ISSUE THREAD

**Issue by glebmiller** [isFakeAccount/psnawp#61]
can you pls tell me, why do I get this error? 

```json
{
    "error": {
        "referenceId": "16dd1d74-b0e7-4d56-aafd-017e15d257f5",
        "code": 3415813,
        "message": "JSON parse error: Cannot construct instance of `com.sony.sie.kamaji.search.universal.model.openapi.SearchDomain`, problem: Unexpected value 'ConceptGameMobileApp'; nested exception is com.fasterxml.jackson.databind.exc.ValueInstantiationException: Cannot construct instance of `com.sony.sie.kamaji.search.universal.model.openapi.SearchDomain`, problem: Unexpected value 'ConceptGameMobileApp'\n at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 1, column: 59] (through reference chain: com.sony.sie.kamaji.search.universal.model.openapi.UniversalSearchRequest[\"domainRequests\"]->java.util.ArrayList[0]->com.sony.sie.kamaji.search.universal.model.openapi.SearchDomainRequest[\"domain\"])",
        "reason": "invalid_parameter",
        "source": "api"
    }
}
```

all I tried was this: 
```py
search_obj = psnawp.search()
title_id_api = search_obj.get_title_id(title_name=trophy_title.title_name) 
```

**Comment by romeo143** [comment]
same problem here

**Comment by isFakeAccount** [comment]
Looks like sony changed something on their end. I'll have to check what are the correct params for the new endpoint.

**Comment by jgavrus** [comment]
Hi, @isFakeAccount , did you have any time to see this problem? or could you send the manual, where I can see what they use now instead `"domain": "ConceptGameMobileApp"`

**Comment by platinumachievements** [comment]
has there been any progress on this? im still getting the same error for the domain

**Comment by isFakeAccount** [comment]
> Hi, @isFakeAccount , did you have any time to see this problem? or could you send the manual, where I can see what they use now instead `"domain": "ConceptGameMobileApp"`

Seems like they have changed the endpoints. Now they are using GraphQL endpoints. This is how it looks like

```
https://m.np.playstation.com/api/graphql/v1/op?operationName=metGetContextSearchResults&variables=%7B%22searchTerm%22%3A%22GTA%22%2C%22searchContext%22%3A%22MobileUniversalSearchGame%22%2C%22displayTitleLocale%22%3A%22en-US%22%7D&extensions=%7B%22persistedQuery%22%3A%7B%22version%22%3A1%2C%22sha256Hash%22%3A%22a2fbc15433b37ca7bfcd7112f741735e13268f5e9ebd5ffce51b85acc126f41d%22%7D%7D
```

**Comment by isFakeAccount** [comment]
Fixed in version 2.0.0
